src/main/java/spleetwaise/transaction/logic/commands/FilterCommand.java

```diff
@@ -36,7 +37,8 @@ public class FilterCommand extends Command {
             COMMAND_WORD + ": Filter the transaction book.\n" + "Parameters: "
                     + "[INDEX] " + "[" + PREFIX_AMOUNT + "AMOUNT] " + "[" + PREFIX_DESCRIPTION
                     + "DESCRIPTION] " + "[" + PREFIX_DATE + "DATE] " + "[" + PREFIX_STATUS
-                    + "STATUS] " + "[" + PREFIX_AMOUNT_SIGN + "AMOUNT_SIGN]\n"
+                    + "STATUS] " + "[" + PREFIX_AMOUNT_SIGN + "AMOUNT_SIGN]"
```

Comment:
```suggestion
                    + "STATUS] " + "[" + PREFIX_AMOUNT_SIGN + "AMOUNT_SIGN] "
```
Should have a space here to accommodate the next prefix

Reply:
Oh thank you thank you